math/mathcore/test/stress/testGenVector.cxx

```diff
@@ -4,6 +4,16 @@
 
 #include "gtest/gtest.h"
 
+#ifndef TYPED_TEST_SUITE_P
+#define TYPED_TEST_SUITE_P TYPED_TEST_CASE_P
+#endif
+#ifndef REGISTER_TYPED_TEST_SUITE_P
+#define REGISTER_TYPED_TEST_SUITE_P REGISTER_TYPED_TEST_CASE_P
+#endif
+#ifndef INSTANTIATE_TYPED_TEST_SUITE_P
+#define INSTANTIATE_TYPED_TEST_SUITE_P INSTANTIATE_TYPED_TEST_CASE_P
+#endif
+
```

Comment:
Can you add a comment `// Backward compatibility for gtest version < xyz`, please (with an appropriate "xyz") so we know in 10 years that we can safely remove this? (Same for the other similar occurrences.)

Reply:
Done.

Reply:
Oh wow, thank you, super nice.